tests/test_basics.py

```diff
@@ -15,6 +15,8 @@ def test_straxen():
 
             print("Downloading test data (if needed)")
             st = straxen.contexts.demo()
+            # Ignore strax-internal warnings
+            st.set_context_config({'free_options': tuple(st.config.keys())})
```

Comment:
Why is this needed for the event display?

Reply:
Not really for this event display but I noticed that we were seeing quite some warnings about things that are really not interesting to be seeing in `pytests`.